torch/nn/modules/linear.py

```diff
@@ -12,16 +13,14 @@ class Linear(Module):
     Args:
         in_features: size of each input sample
         out_features: size of each output sample
-        bias: If set to False, the layer will not learn an additive bias.
-            Default: True
+        bias: If set to False, the layer will not learn an additive bias. Default: True
```

Comment:
I think we're trying to keep the line lengths under 80 chars, so please undo these formatting changes (I also agree they look nicer but hey).